app/models/async_request/job.rb

```diff
@@ -36,15 +34,40 @@ def execution_time
 
     private
 
+    def log_error(log_message, error)
+      Rails.logger.info(log_message)
+      Rails.logger.error "#{error.inspect} \n #{error.backtrace.join("\n")}"
+      Rollbar.error(error, log_message, params: filtered_params(params).inspect)
+    end
+
     def map_status_code(status_code)
       return Rack::Utils::SYMBOL_TO_STATUS_CODE[status_code] if status_code.is_a?(Symbol)
       status_code.to_i
     end
 
     def filtered_params(params)
-      ActionDispatch::Http::ParameterFilter.new(Rails.application.config.filter_parameters)
-                                           .filter(params.compact)
-                                           .inspect
+      return params unless params.is_a?(Array) || params.is_a?(Hash)
+      params.is_a?(Array) ? filter_array(compact_data(params)) : single_filter(compact_data(params))
```

Comment:
compact_data creo que está de más. Porque solo lo estamos llamando si es un array o un hash no?
Lo dejaría así directo:
```suggestion
      params.is_a?(Array) ? filter_array(params.compact) : single_filter(params.compact)
```

Reply:
si, totalmente